test/spec/GeoExt/form/field/GeocoderComboBox.test.js

```diff
@@ -0,0 +1,148 @@
+                expect(geocoderCombo).to.be.an(
+                    GeoExt.form.field.GeocoderComboBox);
+            });
+        });
+    });
+
+    describe('public functions', function() {
+        var geocoderCombo;
+        beforeEach(function() {
+            geocoderCombo = Ext.create('GeoExt.form.field.GeocoderComboBox');
+        });
+
+        it('are correctly defined', function() {
+            expect(typeof geocoderCombo.convertToExtent).to.be('function');
+            expect(typeof geocoderCombo.convertToCoordinate).to.be('function');
+            expect(
+                typeof geocoderCombo.drawLocationFeatureOnMap
+            ).to.be('function');
+            expect(
+                typeof geocoderCombo.removeLocationFeature
+            ).to.be('function');
+        });
+    });
+
+    describe('configs and properties', function() {
+        var geocoderCombo;
+        beforeEach(function() {
+            geocoderCombo = Ext.create('GeoExt.form.field.GeocoderComboBox');
+        });
+
+        it('are correctly defined (with defaults)', function() {
+            expect(geocoderCombo.map).to.be(null);
+            expect(geocoderCombo.displayField).to.be('name');
+            expect(geocoderCombo.displayValueMapping).to.be('display_name');
+            expect(geocoderCombo.valueField).to.be('extent');
+            expect(geocoderCombo.queryParam).to.be('q');
+            expect(geocoderCombo.emptyText).to.be('Search a location');
+            expect(geocoderCombo.minChars).to.be(3);
+            expect(geocoderCombo.queryDelay).to.be(100);
+            expect(geocoderCombo.url).to.be(
+                'https://nominatim.openstreetmap.org/search?format=json');
+            expect(geocoderCombo.srs).to.be('EPSG:4326');
+            expect(geocoderCombo.zoom).to.be(10);
+            expect(geocoderCombo.showLocationOnMap).to.be(true);
+        });
+
+        it('store is created if not configured', function() {
+            expect(
+                geocoderCombo.store instanceof Ext.data.JsonStore
+            ).to.be(true);
+        });
+
+        it('locationLayer is created if not configured', function() {
+            expect(
+                geocoderCombo.locationLayer instanceof ol.layer.Vector
+            ).to.be(true);
+        });
+    });
+
+    describe('event handling', function() {
+        var geocoderCombo;
+        var olMap;
+        beforeEach(function() {
+            olMap = new ol.Map({
+                view: new ol.View({
+                    center: [10, 10],
+                    zoom: 0
+                })
+            });
+            geocoderCombo = Ext.create('GeoExt.form.field.GeocoderComboBox', {
+                map: olMap
+            });
+        });
+
+        describe('select', function() {
+            it('centers the map to coordinate', function() {
+                var record = {
+                    get: function() {
+                        return [0, 0];
```

Comment:
maybe you can change the coordinates to something else, as [0,0] are the defaults of the map already

Reply:
Done, see 0cd0dc59164c4d5a0